client/src/main/scala/drt/client/components/ContactPage.scala

```diff
@@ -181,8 +181,12 @@ object FeedBackComponent {
                       )
                     ),
                     MuiGrid(item = true, xs = 12)(
-                      MuiButton(variant = "outlined", sx = SxProps(Map("border" -> "1px solid white", "color" -> "white", "font-weight" -> "bold")))(
-                        "Give feedback >", ^.onClick --> Callback(dom.window.open(s"${SPAMain.urls.rootUrl}/feedback/contact-us/$aORbTest", "_blank")),
+                      MuiButton(variant = "outlined", sx = SxProps(Map("textTransform" -> "none",
+                        "border" -> "1px solid white",
+                        "color" -> "white",
+                        "font-weight" -> "bold",
+                        "font-size" -> "1.50rem")))(
```

Comment:
We should be using a standard button font size & weight as specified in the theme